packages/desktop-client/src/locale/en.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
**🔨 warning:** ‏language files should not be committed to this repository.

Reply:
File removed and added as an ignore rule.